web/about.html

```diff
@@ -0,0 +1,472 @@
+          helpers to make this an amazing conference. If a new
+          Volunteer Call happens, we will everyone.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="d-md-flex flex-md-equal w-100 ps-md-3 bg-pink px-5">
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+      <div class="carousel-inner">
+        <div class="bg-pink mr-md-3 pt-3 px-3 mx-5 py-md-5 px-md-5 overflow-hidden">
+          <div class="carousel-item active">
+            <div class="row">
+            
+            
+              
+            <div class="col-lg-3">
+              <div class="row">
+                <div class="col-4 d-none d-lg-block">
+                  <img class="rounded-circle img-fluid" src="img/org/naomi.jpg" alt="Generic placeholder image">
+                </div>
+                <div class="col-8 align-start">
+                  <h5 style="position: inline;" class="text-white">Naomi Ceder</h5>
+                  <p class="text-white">Sponsorship Chair</p>
+                </div>
+              </div>
+              <p class="mt-3 text-white">
+                Author of Quick Python Book, former chair of the PSF, using and teaching Python since 2001
+              </p>
+              </div><!-- /.col-lg-4 -->
+            
+            
+              
+            <div class="col-lg-3">
+              <div class="row">
+                <div class="col-4 d-none d-lg-block">
+                  <img class="rounded-circle img-fluid" src="img/org/mariatta.jpg" alt="Generic placeholder image">
+                </div>
+                <div class="col-8 align-start">
+                  <h5 style="position: inline;" class="text-white">Mariatta </h5>
+                  <p class="text-white">Co-Chair</p>
+                </div>
+              </div>
+              <p class="mt-3 text-white">
+                Mariatta is a Python core developer where she focuses on improving the workflow and documentation. She is active in the Python community as an advisor for the Global PyLadies, co-founder of PyCasca...
+              </p>
+              </div><!-- /.col-lg-4 -->
+            
+            
+              
+            <div class="col-lg-3">
+              <div class="row">
+                <div class="col-4 d-none d-lg-block">
+                  <img class="rounded-circle img-fluid" src="img/org/maria.jpg" alt="Generic placeholder image">
+                </div>
+                <div class="col-8 align-start">
+                  <h5 style="position: inline;" class="text-white">Maria Jose</h5>
+                  <p class="text-white">Program Chair</p>
+                </div>
+              </div>
+              <p class="mt-3 text-white">
+                I am a passionate biologist, working as a Sr. Data Scientist in Berlin. In my spare time, I love to develop projects that can help people around, especially people who are eager to learn new things...
```

Comment:
It seems like the organizer bio are getting cut off. Either we ask people to shorten their bio, or adjust the display so everything get shown. But the "..." at the end doesn't look too good :(

Reply:
That'd be lovely. Maybe I will encourage people on slack to submit a fix for their descriptions, so we can update the CSV as well.